### PR TITLE
Update popular extensions list

### LINKED ISSUE #166: Code Folding is still listed under Popular Extensions
Code Folding is still listed under Popular Extensions on main page of brackets.io. This is now a default extension, so it shouldn't appear in this list.

### PR DESCRIPTION
**Pull request by mynetx**
As per Health Report Data Sep 26 to Oct 26, 2015.
Also provide l10n attributes for extension descriptions.
Fix #166.

### CODE REVIEW

**Comment by mynetx** [comment]
image

File: index.html [hunk 1]
```diff
@@ -205,29 +205,26 @@ <h2 data-i18n="index.page.content.new-features.new-features">New Features</h2>
...
+            <p data-i18n="index.page.content.popular-extensions.list.ivogabe-icons">File icons in Brackets’ file tree.</p>
 
-            <h3><a href="https://github.com/zaggino/brackets-git">Brackets Git</a></h3>
-            <p>Git integration for Brackets</p>
+            <h3><a href="https://github.com/lkcampbell/brackets-indent-guides">Indent Guides</a></h3>
+            <p data-i18n="index.page.content.popular-extensions.list.indent-guides">Show indent guides in the code editor.</p>
 
-            <h3><a href="https://github.com/mikaeljorhult/brackets-autoprefixer">Autoprefixer</a></h3>
-            <p>Parse CSS and add vendor prefixes automatically.</p>
+            <h3><a href="https://github.com/zaggino/brackets-git">Git</a></h3>
```
Comment (abose) [inline index.html:220]: `Brackets Git` is the extension name.

**Comment by abose** [comment]
Tagging @ryanstewart @nethip as this is a high visibility change.
+1 for the updated extension list.

**Comment by ryanstewart** [comment]
Yup, this looks good to me! Thanks @jublo!

**Comment by abose** [comment]
Thanks @jublo .
Merging :+1:

**Pull request merged**